### ISSUE TITLE
Using spreadsheetId instead of spreadsheetName

### ISSUE THREAD

**Issue by ddresch** [ongoworks/meteor-google-spreadsheets#12]
First of all thanks for putting together this package. It's pretty handy and I'm using it to manage keys for i18n purpose.

```
EditGoogleSpreadsheet.load({
      //debug: true,
      spreadsheetId: spreadsheetName,
...
```

I needed to change the package to use the spreadsheetId instead of the name. I'm using subfolders in google drive etc. so using simply the name didn't work.  Now I'm wondering if it's not the better solution using the ID anyways?

Not sure if this package is still under development but just in case :)

**Comment by aaronjudd** [comment]
Not under development, but we are actively using it still -  it just works for our case.  Happy to take a PR for this though, likely you could just create a new method based on  `spreadsheet/fetch2` that uses `spreadsheetId` as the key.

**Comment by ddresch** [comment]
Wow quick response, you can check the PR #13 which adds a simple flag to options object. Like this it's possible to use the same method instead of a new one.

**Comment by ddresch** [comment]
Perfect it's published via version 0.5.1
